crates/astria-conductor/Cargo.toml

```diff
@@ -35,11 +35,12 @@ astria-sequencer-client = { path = "../astria-sequencer-client", features = [
 ] }
 astria-sequencer-relayer = { path = "../astria-sequencer-relayer" }
 astria-sequencer-types = { path = "../astria-sequencer-types" }
+astria-sequencer-validation = { path = "../astria-sequencer-validation" }
```

Comment:
I have started writing these like so:
```toml
sequencer-validation = { package = "astria-sequencer-validation", path = "../astria-sequencer-validation" }
```
So that in the rust code you have the slightly shorter `sequencer_validation::FooBar`. You can see this here with the telemetry crate.

I think that's neater, but I should have probably brought it up for discussion.